src/components/forms/AppSearchInput.vue

```diff
@@ -22,6 +39,8 @@ const props = defineProps<{
 
 const searchText = ref(props.modelValue);
 
+const clearField = () => (searchText.value = '');
```

Comment:
I think this needs to `emit('update:modelValue', '')` instead of setting `searchText`, as my understanding of the spec is that it should update the search results too

`searchText` is the internal (unsubmitted) value of the search box, triggering an update on `modelValue` is how to propagate the change out of the component.